src/chessboard/components/Square.tsx

```diff
@@ -51,6 +51,8 @@ export function Square({
     setPromoteFromSquare,
     setPromoteToSquare,
     setShowPromoteDialog,
+    lastSquareDraggedOver,
+    setLastSquareDraggedOver,
```

Comment:
all imports in project are in alphabetical order, not critical for me, but will be nice to keep that order